### ISSUE TITLE
possible incorrect formatting

### ISSUE THREAD

**Issue by ericrudder** [pulumi/docs#2787]
[praneet, this was the issue we discussed in slack on AWS doc]

![image](https://user-images.githubusercontent.com/8387010/77971030-8d5a5f80-72a2-11ea-8ba0-3452e3027ea5.png)

**Comment by praneetloke** [comment]
Yep. This is possibly another instance of https://github.com/pulumi/pulumi-terraform-bridge/issues/113.

**Comment by ekrengel** [comment]
> This is possibly another instance of pulumi/pulumi-terraform-bridge#113.

This one actually is a bit different because these are sub bullets / part of the object that is the higher level bullet.

**Comment by praneetloke** [comment]
Possibly related past issue https://github.com/pulumi/pulumi-terraform-bridge/pull/138.
